storage/simulation/cluster.go

```diff
@@ -195,7 +207,49 @@ func (c *Cluster) runEpoch() bool {
 	// Output the update.
 	c.OutputEpoch()
 
-	return stable
+	// Flush the output.
+	c.flush()
+
+	return lastEpoch
+}
+
+// executeScriptedActions performs all scripted actions to the cluster and
+// return true if the exit action is encountered.
+func (c *Cluster) executeScriptedActions() bool {
+	var lastEpoch bool
+	actions := c.script.getActions(c.epoch)
+	for _, action := range actions {
+		switch action.operation {
+		case (OpExit):
```

Comment:
Do these parens on case statements do anything? If not we should get rid of them.